### PR TITLE
 [Docathon] Fix NO.13-NO.18 API label

### PR DESCRIPTION
**Pull request by zade23**
### PR types

Others

### PR changes

Docs 

### Description

#57363

| 序号 | Origin label name                       | Fixed  label name                             |
| ---- | --------------------------------------- | --------------------------------------------- |
| 13   | `api_base_default_main_program`         | `api_static_default_main_program`             |
| 14   | `api_base_default_startup_program`      | `api_static_default_startup_program`          |
| 15   | `api_base_dygraph_disable_dygraph`      | `api_paddle_enable_static`                    |
| 16   | `api_base_dygraph_enable_dygraph`       | `api_paddle_disable_static`                   |
| 17   | `api_base_dygraph_guard`                | `api_paddle_guard`                            |
| 18   | `api_base_dygraph_to_variable`          | `api_paddle_to_tensor`                        |

### CODE REVIEW

File: python/paddle/optimizer/optimizer.py [hunk 1]
```diff
@@ -1604,7 +1604,7 @@ def minimize(
             loss (Tensor): A ``Tensor`` containing the value to minimize.
             startup_program (Program, optional): :ref:`api_base_Program` for
                 initializing parameters in ``parameters``. The default value
-                is None, at this time :ref:`api_base_default_startup_program` will be used.
+                is None, at this time :ref:`api_static_default_startup_program` will be used.
```
Comment (ooooo-create) [inline python/paddle/optimizer/optimizer.py]:
```suggestion
                is None, at this time :ref:`api_paddle_static_default_startup_program` will be used.
```

File: python/paddle/optimizer/optimizer.py [hunk 2]
```diff
@@ -1243,7 +1243,7 @@ def backward(
             loss (Tensor): ``loss`` tensor to run optimizations.
             startup_program (Program, optional): :ref:`api_base_Program` for
                 initializing parameters in ``parameters``. The default value
-                is None, at this time :ref:`api_base_default_startup_program` will be used.
+                is None, at this time :ref:`api_static_default_startup_program` will be used.
```
Comment (ooooo-create) [inline python/paddle/optimizer/optimizer.py]:
```suggestion
                is None, at this time :ref:`api_paddle_static_default_startup_program` will be used.
```

File: python/paddle/nn/clip.py [hunk 3]
```diff
@@ -965,7 +965,7 @@ def set_gradient_clip(clip, param_list=None, program=None):
                 It can be a list of parameter or a list of parameter's name.
                 Default None, meaning that all parameters in the program will be included.
         program (Program, optional): The program where parameters are located.
-                Default None, meaning that using :ref:`api_base_default_main_program` .
+                Default None, meaning that using :ref:`api_static_default_main_program` .
```
Comment (ooooo-create) [inline python/paddle/nn/clip.py]:
```suggestion
                Default None, meaning that using :ref:`api_paddle_static_default_main_program` .
```

File: python/paddle/incubate/optimizer/modelaverage.py [hunk 4]
```diff
@@ -302,7 +302,7 @@ def minimize(
             loss (Tensor): A ``Tensor`` containing the value to minimize.
             startup_program (Program, optional): :ref:`api_base_Program` for
                 initializing parameters in ``parameters``. The default value
-                is None, at this time :ref:`api_base_default_startup_program` will be used.
+                is None, at this time :ref:`api_static_default_startup_program` will be used.
```
Comment (ooooo-create) [inline python/paddle/incubate/optimizer/modelaverage.py]:
```suggestion
                is None, at this time :ref:`api_paddle_static_default_startup_program` will be used.
```

File: python/paddle/incubate/optimizer/lookahead.py [hunk 5]
```diff
@@ -252,7 +252,7 @@ def minimize(
             loss (Tensor): A ``Tensor`` containing the value to minimize.
             startup_program (Program, optional): :ref:`api_base_Program` for
                 initializing parameters in ``parameters``. The default value
-                is None, at this time :ref:`api_base_default_startup_program` will be used.
+                is None, at this time :ref:`api_static_default_startup_program` will be used.
```
Comment (ooooo-create) [inline python/paddle/incubate/optimizer/lookahead.py]:
```suggestion
                is None, at this time :ref:`api_paddle_static_default_startup_program` will be used.
```

File: python/paddle/distributed/fleet/fleet.py [hunk 6]
```diff
@@ -1254,7 +1254,7 @@ def minimize(
             loss (Tensor): A ``Tensor`` containing the value to minimize.
             startup_program (Program, optional): :ref:`api_base_Program` for
                 initializing parameters in ``parameter_list``. The default value
-                is None, at this time :ref:`api_base_default_startup_program` will be used.
+                is None, at this time :ref:`api_static_default_startup_program` will be used.
```
Comment (ooooo-create) [inline python/paddle/distributed/fleet/fleet.py]:
```suggestion
                is None, at this time :ref:`api_paddle_static_default_startup_program` will be used.
```

File: python/paddle/base/dygraph/base.py [hunk 7]
```diff
@@ -159,9 +159,9 @@ def _convert_into_variable(tensor):
 def enabled():
     """
     This function checks whether the program runs in dynamic graph mode or not.
-    You can enter dynamic graph mode with :ref:`api_base_dygraph_guard` api,
-    or enable and disable dynamic graph mode with :ref:`api_base_dygraph_enable_dygraph`
-    and :ref:`api_base_dygraph_disable_dygraph` api .
+    You can enter dynamic graph mode with :ref:`api_paddle_guard` api,
```
Comment (ooooo-create) [inline python/paddle/base/dygraph/base.py]: @sunzhongkai588 因为2.0默认是动态图模式，所以把 <You can enter dynamic graph mode with :ref:`api_base_dygraph_guard` api,> 删了，修改成 <You can enable and disable dynamic graph mode with :ref:`api_base_dygraph_enable_dygraph` ...> ,孙师傅看看怎么样~

**Review by ooooo-create — COMMENTED** [review]
忘记加上paddle啦~

File: python/paddle/base/dygraph/base.py (hunk 7 above)
Comment (sunzhongkai588) [reply python/paddle/base/dygraph/base.py]: 我觉得 ok 的， @SigureMo 看一下

**Review by sunzhongkai588 — DISMISSED** [review]
LGTM

File: python/paddle/base/dygraph/base.py (hunk 7 above)
Comment (SigureMo) [reply python/paddle/base/dygraph/base.py]: `paddle.base.dygrapph.enable_dygraph` 不是暴露的 API 吧？应该引用不了吧

- `enable_dygraph` == `disable_static`
- `disable_dygraph` == `enable_static`

可以用 `paddle.enable_static` 和 `paddle.disable_static` 这两个 API

Comment (ooooo-create) [reply python/paddle/base/dygraph/base.py]: 是的，我引用了修改之前的文字内容😂，已经修改过了

Comment (ooooo-create) [reply python/paddle/base/dygraph/base.py]: @zade23 可以修改下，enter dynamic graph mode with :ref:`api_paddle_guard` api,删了变成 you can enable and disable dynamic graph mode with :ref:`api_paddle_disable_static`
    and :ref:`api_paddle_enable_static` api .

Comment (zade23) [reply python/paddle/base/dygraph/base.py]: > @zade23 可以修改下，enter dynamic graph mode with :ref:`api_paddle_guard` api,删了变成 you can enable and disable dynamic graph mode with :ref:`api_paddle_disable_static` and :ref:`api_paddle_enable_static` api .

@ooooo-create 
了解了，如果保持意思不变，改成如下这样会不会更通顺一点呢🤔
```py
You can enable dynamic graph mode with :ref:`api_paddle_disable_static` api, 
or disable dynamic graph mode with :ref:`api_paddle_enable_static` .
```

**Review by sunzhongkai588 — APPROVED** [review]
LGTM

**Pull request merged**